<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emplyee</class>
 <widget class="QDialog" name="emplyee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/imagestart/pic/logs1.jpg);</string>
  </property>
  <widget class="QLineEdit" name="cpassworde2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/imagestart/pic/t1.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;MV Boli&quot;;
background-image:url(:/imagestart/pic/t2.jpg);</string>
   </property>
   <property name="text">
    <string>add employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="passworde">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image:url(:/imagestart/pic/t2.jpg);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passworde2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/imagestart/pic/t1.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image:url(:/imagestart/pic/t2.jpg);</string>
   </property>
   <property name="text">
    <string>Account Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpassworde">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image:url(:/imagestart/pic/t2.jpg);</string>
   </property>
   <property name="text">
    <string>Confrim Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namee">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/imagestart/pic/t1.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image:url(:/imagestart/pic/t2.jpg);</string>
   </property>
   <property name="text">
    <string>Full Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernamee">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/imagestart/pic/t1.jpg);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
